tox.ini

```diff
@@ -45,20 +34,11 @@ commands=codecov -X gcov --file {toxworkdir}/coverage.xml
 
 [testenv:docs]
 extras =
-        docs
-        testing
+	docs
+	testing
 changedir = docs
 commands =
-        {envpython} -m sphinx \
-          -j auto \
-          -b html \
-          --color \
-          -a \
-          -n \
-          -W \
-          -d "{temp_dir}/.doctrees" \
```

Comment:
keeping this cache allows faster rebuilds

Reply:
I'm not too worried about this. Seems like another premature optimization.